src/gui/tray/asyncimageresponse.cpp

```diff
@@ -17,7 +17,6 @@
 #include <QSvgRenderer>
 
 #include "asyncimageresponse.h"
-#include "usermodel.h"
```

Comment:
I think replacing this include with `#include "accountmanager.h"` will make the code build again -- should be enough to fix the references to `OCC::AccountPtr` and `OCC::AccountManager`

Reply:
Strange, was building fine locally

I fixed it now, thanks